minecraft_servers/mapfantasy/manifest.json

```diff
@@ -0,0 +1,37 @@
+{
+  "server_name": "mapfantasy",
+  "nice_name": "MapFantasy",
+  "direct_ip": "mapfantasy.eu",
+  "social": {
+    "web": "https://mapfantasy.eu",
+    "discord": "https://mpfntsy.link/discord",
```

Comment:
As custom discord invite links are only supported for LabyMod partners at the moment.
Please use a discord invite link.

Documentation reference: https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#discord-url